.semaphore/semaphore.yml

```diff
@@ -379,3 +379,5 @@ promotions:
     pipeline_file: validate-sidecar-version.yml
     auto_promote:
       when: "result = 'passed' and change_in(['/.versions/ide-sidecar.txt', '/src/clients/sidecar-openapi-specs/sidecar.openapi.yaml'], {default_branch: 'main', branch_range: '$SEMAPHORE_GIT_COMMIT_RANGE', pipeline_file: 'ignore'})"
+  - name: CCloud Auth Smoke Tests
+    pipeline_file: ccloud-auth-smoketests.yml
```

Comment:
TODO: remove this

Reply:
Removed in https://github.com/confluentinc/vscode/pull/2292/commits/21ccab47559eba7f2a4afa0c0bcc1e8548f36a4f